system/jlib/jevent.cpp

```diff
@@ -841,21 +862,9 @@ void EventRecorder::writeByte(offset_type & offset, byte value)
 void EventRecorder::writeBlock(offset_type startOffset, size32_t size)
 {
     //MORE: Make this asynchronous - on a background thread.
-    offset_t fileOffset = startOffset & ~blockMask;
```

Comment:
This check was not good enough - chechDataWrite is now correct.  In the future I will look at blocking to avoid the potential for a corrupt file.